tdr-admins/tdr-user-administrator.md

```diff
@@ -61,11 +61,10 @@ If a new user needs to be added, then:
     * Email
 4. In the "Required User Actions" add the following options:  
   * Configure OTP: *(this will enforce MFA)*
-    * **Note**: For the moment, do not configure OTP (one-time password). The TDR team are currently working out which multi-factor authentication methods will be supported.
 5. Click "save"
 6. Under the "Credentials" tab: ![](images/tdr-user-administrator/adding_new_tb_user/users_3_v2.png)
```

Comment:
Could you update the screenshot to show the `Configure OTP (CONFIGURE_TOTP)` option being set: `images/tdr-user-administrator/adding_new_tb_user/users_3_v2.png`

Reply:
I've updated it and saved it as `users_3_v3.png`